Source/WebKit/ChangeLog

```diff
@@ -1,3 +1,14 @@
+2022-05-09  Yousuke Kimoto  <yousuke.kimoto@sony.com>
+
+        [WinCairo] Fix a compile error due to InspectorResourceURLSchemeHandler
+        https://bugs.webkit.org/show_bug.cgi?id=239490
+
+        Reviewed by NOBODY (OOPS!).
+
+        No new tests (OOPS!).
```

Comment:
Remove this line.

Reply:
Thanks, the line was deleted.

Reply:
@Constellation ,

I haven't noticed you approved this PR, and I asked you review this PR again, sorry.

Reply:
np!
Unfortunately, you need to squash commits into one right now. But the other one looks good to me.